irohad/validation/stateful/impl/stub_validator.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <validation/stateful/stub_validator.hpp>
+#include <dao/dao.hpp>
+#include <numeric>
+#include <validation/stateful/stub_command_validator.hpp>
+
+namespace iroha {
+  namespace validation {
+
+    /**
+     * Interface for performing stateful validation
+     */
+    dao::Proposal ValidatorStub::validate(const dao::Proposal &proposal,
+                                          ametsuchi::TemporaryWsv &wsv) {
+      auto command_validator = CommandValidatorStub(wsv);
+      auto
+          checking_transaction =
+          [&command_validator](auto &tx, auto &executor, auto &query) {
```

Comment:
Should add a line that
`// TODO permission validation`.
Becouse permission validation needs tx.signatures.